server/routes.js

```diff
@@ -17,6 +17,11 @@ const routes = function(app) {
 		res.sendFile(path.join(__dirname, '..', 'built', 'shrdlu.html'));
 	});
 
+	// environment variables set in .env that the frontend might need
```

Comment:
This approach with env.js and use of the BASE_URL variable attached to the global was meant to allow us to include variables from the .env file in the frontend.  We ended up going a different route and this is removed in a future checkin